tests/dogtag/pytest-ansible/.gitlab-ci.yml

```diff
@@ -1,9 +1,9 @@
 image: bbhavsar/dogtagpki-test-essentials-container
 
 variables:
-  HOSTFILE: $CI_PROJECT_DIR/linchpin-openstack/inventories/openstack_topo.inventory
-  IMG_BRANCH: fedora28_1master
-  POST_PROV: 'https://gitlab.cee.redhat.com/snippets/393/raw'
+  HOSTFILE: $CI_PROJECT_DIR/tests/dogtag/pytest-ansible/provision/ansible_inventory
+  IMG_NAME: 'Fedora 31'
```

Comment:
Is `Fedora 31` a valid image name? I'm asking this because I'm concerned with the `F` and the space. Usually the image names are all lower case and no spaces.

Reply:
Yes Fedora 31 is real image name .!